compiler-rt/lib/sanitizer_common/sanitizer_common_interceptors.inc

```diff
@@ -10655,7 +10554,6 @@ static void InitializeCommonInterceptors() {
   INIT_MD5;
```

Comment:
Should we remove md5?

From my experience interceptors, other then alloc, mem* str*, detect very few issues, however require some maintaince.
Some interceptors are unavoidable to maintan state, e.g. signals, pthread etc. For almost msan interceptors are needed. However for asan they are mostly optional.

So looking back my opinion we should rather avoid interceptor if we can.